src/config.ts

```diff
@@ -79,6 +79,13 @@ function initializeConfig(): Config {
       configInstance.defaultFirstTag = configInstance.defaultFirstTag.substring(1);
     }
 
+    // Validate module ref mode
+    if (!ALLOWED_MODULE_REF_MODES.includes(configInstance.moduleRefMode as (typeof ALLOWED_MODULE_REF_MODES)[number])) {
```

Comment:
Can you see if we need the typeof here? It should be derived, no? If possible, let's remove it.